packages/react/readme.md

```diff
@@ -1,4 +1,90 @@
-
 # Hops React
 
-T.B.D. (To Be Documented)
+hops-react works in tandem with hops-cli and hops-config to make an integrated solution for universal ("isomorphic") rendering using React. It provides a minimal API and hides the tricky bits of setting up React for such use-cases.
```

Comment:
I'd link hops-cli and hops-config to the respective readme on Github or npmjs.com